spec/models/invoice_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Invoice, type: :model do
+  describe 'DB table' do
+      it { is_expected.to have_db_column :date }
+      it { is_expected.to have_db_column :due_date }
```

Comment:
please test all of the columns that you have for the invoice model

Reply:
fixed